docs/user_guides/fs/transformation_functions.md

```diff
@@ -94,8 +103,68 @@ The creation of a many-to-many transformation function is similar to that of a o
         import pandas as pd
 
         @udf(return_type=[int, int, int])
+        def add_one_multiple(feature1, feature2, feature3):
+            return feature1 + 1, feature2 + 1, feature3 + 1
+        ```
+
+### Specifying execution modes
+
+The `mode` parameter of the `@udf` decorator can be used to specify the execution mode of the transformation function. 
+
+#### Default 
+This execution mode assumes that the transformation function is vectorized and can be executed as both a Pandas UDF and a Python UDF. It serves as the default mode used when the `mode` parameter is not specified. In this mode, the transformation function is executed as a Pandas UDF during training and in the batch inference pipeline, while it operates as a Python UDF during online inference.
+
+
+=== "Python"    
+    !!! example "Creating a many to many transformations function using the default execution mode"
```

Comment:
Can you explain about what gets dropped here?
What are the names of the input columns of the DF and the names of the output columns?

Reply:
I removed the drop parameter for these examples, since it is explained later on here : https://github.com/logicalclocks/logicalclocks.github.io/pull/409/files#diff-416b7f662fbf20e660e151b36535f28c659c927f61e790ac25eb3a40a3638b25R172.

Reply:
Do you think that this explanation is clear enough or does it require more be more in detail?